Main.hs

```diff
@@ -4,23 +4,40 @@ import Control.Monad (unless)
 import Text.Printf (printf)
 
 short :: [a] -> Bool
-short = undefined 
+short (a : b : c : xs) = False
```

Comment:
No need to introduces names you have no intention of using. Instead opt for wildcards.